docs/concepts/components/refs.md

```diff
@@ -25,3 +25,6 @@ html! {
 // In rendered
 let has_attributes = self.node_ref.cast::<Element>().unwrap().has_attributes();
 ```
+
+## Relevant Example
```

Comment:
Because it's a list (and people might add to it) I think it's OK to use "examples" instead of "example"
```suggestion
## Relevant examples
```